scout/mseg_meta.py

```diff
@@ -417,11 +417,11 @@ def import_commercial_time_preference_data(file_name):  # kprem.txt
         files_str = ''
         for file_ in files_:
             if file_ != files_[-1]:
-                files_str = files_str + file_ + ', '
+                files_str = files_str + str(file_) + ', '
             elif len(files_) == 1:
-                files_str = files_str + file_
+                files_str = files_str + str(file_)
             else:
-                files_str = files_str + 'and ' + file_
+                files_str = files_str + 'and ' + str(file_)
```

Comment:
This is fine, but I do think f-strings are cleaner (e.g. `f"{files_str}{str(file_)}, "`)